dependencies.py

```diff
@@ -41,16 +41,38 @@
 ]
 INTERESTING_IMPORTS = ["numpy", "gdal", "setuptools"]
 
-our_dir = Path(__file__).parent
-PROFILE_LIBRARY_DIR = our_dir.parent.parent
+OUR_DIR = Path(__file__).parent
 
 logger = logging.getLogger(__name__)
 
 
 def ensure_everything_installed():
     """Check if DEPENDENCIES are installed and install them if missing."""
     missing = _check_presence(DEPENDENCIES)
-    _install_dependencies(missing)
+    target_dir = _dependencies_target_dir()
+    _install_dependencies(missing, target_dir=target_dir)
+
+
+def _dependencies_target_dir(our_dir=OUR_DIR):
+    """Return python dir inside our profile
+
+    Return two dirs up if we're inside the plugins dir. If not, we have to
+    import from qgis (which we don't really want in this file) and ask for our
+    profile dir.
+
+    """
+    if "plugins" in str(our_dir).lower():
+        # Looks like we're in the plugin dir. Return ../..
+        return OUR_DIR.parent.parent
+    # We're somewhere outside of the plugin directory. Perhaps a symlink?
+    # Perhaps a development setup? We're forced to import qgis and ask for our
+    # profile directory, something we'd rather not do at this stage. But ok.
+    print("We're not in our plugins directory: %s" % our_dir)
+    from qgis.core import QgsApplication
+
+    python_dir = Path(QgsApplication.qgisSettingsDirPath()) / "python"
+    print("We've asked qgis for our python directory: %s" % python_dir)
```

Comment:
Is there a reason why we use `print` instead of the `logger`?

Reply:
Zoals zonet gezegd: logging is dan nog niet opgezet.